frontend/src/app/event/widgets/event-card/event-card.widget.html

```diff
@@ -0,0 +1,16 @@
+<mat-card [ngClass]="(selected && disableLink) ? 'selected-card' : ''" appearance="outlined"
+    style="margin-top: 12px !important;" [routerLink]="disableLink ? null : '/events/' + event.id"
+    (click)="cardClicked()"
+>
+    <div style="display: flex; flex-direction: column; padding: 12px;">
+        <div style="display: flex; flex-direction: row;">
+            <p style="font-weight: 500; margin: 0px; width: 75%;">{{event.name}}</p>
+            <p style="margin: 0px; text-align: end; width: 25%;">{{event.time | date:'shortTime'}}</p>
+        </div>
+        <div style="display: flex; flex-direction: row; margin-top: 8px;">
+            <p style="margin: 0px; width: 50%;">{{event.organization.shorthand}}</p>
+            <p style="margin: 0px; text-align: end; width: 50%;">{{event.location}}</p>
+        </div>
+        <p class="description-text">{{event.description}}</p>
```

Comment:
I wonder if we should drop the description on the card or have a very short version of it (perhaps first line only?). It's currently unclear what benefit exists in selecting an event from the main list at the moment.

Reply:
@KrisJordan I am actually in support of just dropping it for right now! I agree that there is a bit of a redundancy here. I will commit this change, and we can always add it again later if needed.